osu.Game/Online/Spectator/SpectatorClient.cs

```diff
@@ -35,19 +35,28 @@ public abstract class SpectatorClient : Component, ISpectatorClient
         /// </summary>
         public abstract IBindable<bool> IsConnected { get; }
 
-        private readonly List<int> watchingUsers = new List<int>();
+        /// <summary>
+        /// The states of all users currently being watched.
+        /// </summary>
+        public IBindableDictionary<int, SpectatorState> WatchingUserStates => watchingUserStates;
```

Comment:
Same here, xmldoc says "watched" user states, while name is mentioning "watching" user states.